docs/manual/interpreters.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: "Interpreters"
+description: ""
+group: manual
+---
+{% include JB/setup %}
+
+
+## Interpreters in zeppelin
+
+This section explain the role of Interpreters, interpreters group and interpreters settings in Zeppelin.
+Zeppelin interpreter concept allows any language/data-processing-backend to be plugged into Zeppelin.
+Currently Zeppelin support many interpreters such as Scala(with Apache Spark), Python(with Apache Spark), SparkSQL, Hive, Markdown and Shell.
+
+### What is zeppelin interpreter?
+
+Zeppelin Interpreter is the plug-in which enable zeppelin user to use a specific language backend. For example to use scala code in Zeppelin, you need scala interpreter.
```

Comment:
Both fixed. Thanks for the review!
Eran

On Thu, Aug 27, 2015 at 2:19 AM Felix Cheung <notifications@github.com>
wrote:

> In docs/manual/interpreters.md
> <https://github.com/apache/incubator-zeppelin/pull/256#discussion_r38047152>
> :
>
> > +title: "Interpreters"
> > +description: ""
> > +group: manual
> > +---
> > +{% include JB/setup %}
> > +
> > +
> > +## Interpreters in zeppelin
> > +
> > +This section explain the role of Interpreters, interpreters group and interpreters settings in Zeppelin.
> > +Zeppelin interpreter concept allows any language/data-processing-backend to be plugged into Zeppelin.
> > +Currently Zeppelin support many interpreters such as Scala(with Apache Spark), Python(with Apache Spark), SparkSQL, Hive, Markdown and Shell.
> > +
> > +### What is zeppelin interpreter?
> > +
> > +Zeppelin Interpreter is the plug-in which enable zeppelin user to use a specific language backend. For example to use scala code in Zeppelin, you need scala interpreter.
>
> Is this confusing? there isn't a "Scala" interpreter - there is only a
> Spark interpreter on the list
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-zeppelin/pull/256/files#r38047152>.
>